build.gradle

```diff
@@ -221,6 +222,9 @@ dependencies {
   functionalTestImplementation sourceSets.main.runtimeClasspath
 
   smokeTestImplementation sourceSets.main.runtimeClasspath
+
+  runtime group: 'org.postgresql', name: 'postgresql', version: '42.2.19'
```

Comment:
Is this line required given line 206?

Reply:
Will fix

Reply:
Fixed in https://github.com/hmcts/cpo-case-payment-orders-api/pull/16/commits/caada6f64dfaa5a02352eb768ee9792564c98ff1, deleted line 206